.nais/redis/dev.json

```diff
@@ -0,0 +1,5 @@
+{
+  "plan": "hobbyist",
```

Comment:
Dette er den minste og billigste varianten som burde være tilstrekkelig for dev.